pkg/services/cosmosdb/common-provision.go

```diff
@@ -57,11 +95,48 @@ func (c *cosmosAccountManager) preProvision(
 }
 
 func (c *cosmosAccountManager) buildGoTemplateParams(
+	pp *provisioningParameters,
 	dt *cosmosdbInstanceDetails,
 ) map[string]interface{} {
 	p := map[string]interface{}{}
 	p["name"] = dt.DatabaseAccountName
 	p["kind"] = "GlobalDocumentDB"
+
+	if pp.IPFilterRules != nil {
+		filters := []string{}
+		if pp.IPFilterRules.AllowAzure != "disable" {
+			filters = append(filters, "0.0.0.0")
+		} else if pp.IPFilterRules.AllowPortal != "disable" {
+			// Azure Portal IP Addresses per:
+			// https://aka.ms/Vwxndo
+			// Region	IP address
+			// All regions except those specified below
```

Comment:
I don't understand this... it says all regions _except those specified below_, but _all_ the IPS in the comment appear in the list of filters. This comment needs some formatting fixes, too.

Reply:
Sorry, formatting messed this up. 

All Regions (except german, china, us gov) are covered by: 
40.76.54.131, 52.176.6.30, 52.169.50.45, 52.187.184.26

Germany is 51.4.229.218
China is 139.217.8.252
US Gov is 52.244.48.71

I added them *all* to the list since we don't readily have access to the cloud we're deployed into at this point.  The comment was mostly to document which is which, so i'll reformat it into maybe a tablular format?

Reply:
This is also reproducing the behavior you get in the portal. You can allow access to all azure OR you can allow access only to the portal IPs if you've restricted access to some other list of IPs.

Reply:
Got it. So just fix formatting, I guess.